File: Front-End/Enviro-Quiz/src/app/models/DTO/Applicant.DTO.ts
import {AssessmentDTO} from "./Assessment.DTO";
import {AnswerDTO} from "./Answer.DTO";
import {ResultDTO} from "./ResultDTO.model";

export class ApplicantDTO{
  uuid : string;

  firstName : string;

  lastName : string;

  start : Date = null ;

  end : Date = null ;

  assessmentDTO : AssessmentDTO ;

  answerDTO : AnswerDTO ;

  resultDTO : ResultDTO ;
}

File: BackEnd/Enviro-Quiz/src/main/java/com/enivro/quiz/dto/ApplicantDTO.java
package com.enivro.quiz.dto;


import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;
import java.time.ZonedDateTime;

@Getter
@Setter
public class ApplicantDTO {

    private String uuid ;

    private String firstName ;

    private String lastName ;

    private ZonedDateTime start ;

    private ZonedDateTime end ;

    private AssessmentDTO assessmentDTO ;

    private AnswerDTO answerDTO ;

    private ResultDTO resultDTO ;

}
